### DISCUSSION TITLE
How create a personal branche of a project, Beginner help

### DISCUSSION THREAD

**Discussion by spilz87** [community/community#22572] [New to GitHub]
<p>hello,</p>
<p>I’m so sorry, I think it was already explain somewhere, but I new as Git user and I don’t even know what key word should I use for a search.</p>
<p>The context :</p>
<p>there is a project on Github that I follow, there are a lot of update monthly on this project.</p>
<p>I want to modify some small part to adapt this project to my use, but my modifications are not interesting for other (but I’m totally ok to share them).</p>
<p>Is it possible to have my own branch of the project, with my modifications, but still have the updates of the original project (and just work on the conflicts ) ?</p>
<p>the thing is that I don’t want to merge my modifications to the main project (because the main project doesn’t need them and doesn’t want them), I want to do the opposite, that’s what I don’t understand how to do.</p>
<p>thanks for your help and sorry for my basic question.</p>
<p>regards</p>
<p>Spilz</p>

**Comment by Wabri** [comment, marked as answer]
<p>You can fork the repository and in case of new updates you can merge into your fork</p>

**Comment by spilz87** [comment]
<p>Thanks for you reply, now I understand better it and know the keywords</p>
<p>Regards</p>
